Spanner/src/Transaction.php

```diff
@@ -631,6 +647,18 @@ public function isRetry()
         return $this->isRetry;
     }
 
+    /**
+     * Return this transaction tag.
```

Comment:
```suggestion
     * Return the transaction tag.
```

Additionally, public methods require a sample.